runtime/queries/nix/injections.scm

```diff
@@ -1,153 +1,193 @@
 ((comment) @injection.content
- (#set! injection.language "comment"))
+  (#set! injection.language "comment"))
 
-((comment) @injection.language
-  . ; this is to make sure only adjacent comments are accounted for the injections
-  [
-    (string_expression (string_fragment) @injection.content)
-    (indented_string_expression (string_fragment) @injection.content)
-  ]
-  (#gsub! @injection.language "/%*%s*([%w%p]+)%s*%*/" "%1")
-  (#set! injection.combined))
+((comment) @injection.language.
```

Comment:
Where does the trailing dot come from (again, breaking the query)? 🤔

Reply:
I ran the code with the non-updated query :facepalm: 
Will push the fix

Reply:
Looks good now!